priv/dispatch/dispatch

```diff
@@ -10,6 +10,9 @@
   % Generic endpoint for a 'serialization' and (possibly multiple) 'ontology' as query arguments
   {rdf_any, ["rdf", serialization, id], controller_rdf, []},
 
-  % Generic endpoint for a 'serialization' and (possibly multiple) 'ontology' as query arguments
-  {rdf_any, ["rdf", id], controller_rdf, []}
+  % Generic endpoint for an ID, requiring serialization as query argument
+  {rdf_any, ["rdf", id], controller_rdf, []},
+
+  % Generic endpoint, requiring serialization and (at least) one 'id' as query arguments
```

Comment:
It could be nice to return an rdf descriptive of the website or topic of the website when no id is given - increasing discoverability of what ids to ask for.